.changeset/cli-package-v1.md

```
@@ -0,0 +1,17 @@
+---
+"@evlog/cli": minor
+---
+
+Add `@evlog/cli` — CLI observability for evlog.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)